GCSViews/CameraView.cs

```diff
@@ -331,52 +375,66 @@ private void SetCrosshairType(CrosshairsType type)
 
         #endregion
 
-        private void DoPhoto()
+        private void DoPhoto(string path = null)
```

Comment:
Kép készítési funkcionalitás most éppen szét van szórva e kozott és a CameraHandler között.Mivel a GStreamer itt van megvalósítva, az utóbbira nincs szükség, és a kontroller-en a gomb megnyomásakor lefutó metódust is át kell fűzni erre, illetve akkor itt kell lennie a fájlnév összeállításnak is.

Reply:
Camerahandlertől jön event, mivel a gstreamer miatt a logika a CameraView-ba kellett kerüljön így csak a hívás történik a Handleren keresztül ami miatt elvileg nem kell átfűzni a gomb vezérlést